src/Tribe/Main.php

```diff
@@ -562,19 +562,50 @@ protected function addHooks() {
 			add_filter( 'tribe_settings_url', array( $this, 'tribe_settings_url' ) );
 
 			// Setup Help Tab texting
-			add_filter( 'tribe_help_text_featurebox', array( $this, 'add_help_tab_feature_text' ) );
+			add_filter( 'tribe_help_section_feature-box', array( $this, 'filter_feature_box_content' ) );
+			add_filter( 'tribe_help_section_support', array( $this, 'filter_support_content' ) );
 		}
 
 		/**
 		 * Append the text about The Events Calendar to the feature box on the Help page
 		 *
-		 * @filter tribe_help_text_featurebox
-		 * @param string $text The previous Text
+		 * @filter "tribe_help_section_{$section_id}"
+		 * @param object $section
+		 * @return object $section
+		 */
+		public function filter_feature_box_content( $section ) {
+			$link = '<a href="http://m.tri.be/18j8" target="_blank">' . esc_html__( 'New User Primer', 'the-events-calendar' ) . '</a>';
+
+			$section->content[] = sprintf( __( 'We are committed to helping make your calendar spectacular and have a wealth of resources available to help you out, including a handy %s that will help you get your calendar up and running.', 'the-events-calendar' ), $link );
+
+			return $section;
+		}
+
+		/**
+		 * Append the text about The Events Calendar to the support section on the Help page
 		 *
-		 * @return string
+		 * @filter "tribe_help_section_{$section_id}"
+		 * @param object $section
+		 * @return object $section
 		 */
-		public function add_help_tab_feature_text( $text = '' ) {
-			return $text . ' ' . esc_html__( 'We are committed to helping make your calendar spectacular.', 'the-events-calendar' );
+		public function filter_support_content( $section ) {
+			$section->content[14] = '<strong>' . esc_html__( 'Support for The Events Calendar', 'the-events-calendar' ) . '</strong>';
+
+			$section->content[15] = array(
```

Comment:
Same question applies about the numeric indexes that I called out [in your tribe-common PR](https://github.com/moderntribe/tribe-common/pull/22/files#r43942212).